src/core/data/particles/particle.hpp

```diff
@@ -54,8 +54,8 @@ struct Particle
 
     Particle() = default;
 
-    double weight;
-    double charge;
+    double weight = 0;
```

Comment:
what would it take to not default init particles?

Reply:
a new PR :)